README.md

```diff
@@ -9,3 +9,36 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+## Answer 
+Example:
+- Based on the definition of isomorphism this means that the structure of the edges and non-edges is mainstained under the maping of $f$.
+- In order to prove this we can use a direct proof by explicitly verifying the conditions for isomorphism.
+- $G_{1}$ has vertices $A, B, C$ and one edge $(A, B)$.
+- $G_{2}$ has vertices $1, 2, 3$ and one edge $(1, 2)$.
+- Neither $G_{1}$ or $G_{2}$ are completely connected.
+- If we have a function $f: V_{1} \rightarrow V_{2}$. That maps the vertices and edges form $G_{1}$ to $G_{2}$ such as $f(A) = 1, f(B) = 2, f(C) = 3$.
+- We can look at the correcsponding nodes and edges to check if the to determine if the graph holds the same structure.
+- In $G_{1}, (A, B) \in E_{1}$. $f$ maps to $f(A), f(B) = (1, 2) \in E_{2}$.
+  - The non-edge in $G_{1}$, between $A$ and $C$. Maps to no edge between $f(A) = 1, f(C) = 3$ in $G_{2}$.
+- from here we can see that all edges and non-edges in $G_{1}$ maintain the structue when mapped to the edges and non-edges in $G_{2}$.
+- This satisfies the isomorphism condition.
+
+Counter-example:
```

Comment:
What exactly is this a counter example for? Are you saying that the statement is not true?

Reply:
Well, I really just added it because I felt like it would provide further demonstration of  the conditions for isomorphism. This doesn't change the fact that though our example we demonstrate that we can have two graphs that are not completely connected but still pass the conditions of being isomorphic.

Reply:
No, not necessarily. I only added it because I felt like it helped me understand the conditions for isomorphism better. I Think it worked and can take it out to make the proof more concise.

Reply:
Please don't include anything that's not related to the proof.